packages/grid/_modules_/grid/components/GridCell.tsx

```diff
@@ -115,14 +119,16 @@ export const GridCell: React.FC<GridCellProps> = React.memo((props) => {
       ref={cellRef}
       className={cssClasses}
       role="cell"
+      data-rowid={rowId}
```

Comment:
If you don't need the id, I would recommend we remove this attribute until a developer come up with a good problem it would solve:

- First, the more API we add without real-life problems, is more API that might be useless (that we think we need but nobody really needs). A public API that could make future changes harder.
- Second, It's not sound. It won't retain the type difference between a string and a number. 

If it's for debugging purposes, then it's a different.